wls-vorfaelleundvorkommnisse-service/src/test/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisControllerIntegrationTest.java

```diff
@@ -0,0 +1,170 @@
+                expectedSavedEreignisse.add(ereignis);
+            }
+            Assertions.assertThat(savedEreignisse.size()).isEqualTo(expectedSavedEreignisse.size());
+            Assertions.assertThat(savedEreignisse).filteredOn(ereignis -> ereignis.getClass().equals(Ereignis.class))
+                    .filteredOn(ereignis -> ereignis.getWahlbezirkID().equals(wahlbezirkID));
+        }
+
+        @Test
+        @WithMockUser(authorities = { Authorities.SERVICE_POST_EREIGNISSE, Authorities.REPOSITORY_DELETE_EREIGNISSE, Authorities.REPOSITORY_WRITE_EREIGNISSE })
+        void should_override_old_list_of_ereignis_entities_when_new_data_successfully_saved() throws Exception {
+            // create ereignisse
+            val wahlbezirkID = "wahlbezirkID";
+            List<EreignisModel> ereignisModelList = new ArrayList<>();
+            ereignisModelList.add(TestdataFactory.createEreignisModelWithData());
+            val ereignisseWriteModel = TestdataFactory.createEreignisseWriteModelWithData(wahlbezirkID, ereignisModelList);
+            List<Ereignis> ereignisListToOverride = TestdataFactory.createEreignisEntityListFromModel(ereignisseWriteModel);
+            // save ereignisse
+            ereignisRepository.saveAll(ereignisListToOverride);
+            // read ereignisse
+            SecurityUtils.runWith(Authorities.REPOSITORY_READ_EREIGNISSE);
+            val savedEreignisseBeforeOverridden = ereignisRepository.findByWahlbezirkID(wahlbezirkID);
+            Assertions.assertThat(savedEreignisseBeforeOverridden.size()).isEqualTo(ereignisListToOverride.size());
+
+            // create new ereignisse
+            List<EreignisDTO> ereignisDtoList = new ArrayList<>();
+            ereignisDtoList.add(TestdataFactory.createEreignisDtoWithData());
+            ereignisDtoList.add(TestdataFactory.createEreignisDtoWithData());
+            ereignisDtoList.add(TestdataFactory.createEreignisDtoWithData());
+            val ereignisseWriteDto = TestdataFactory.createEreignisseWriteDTOWithData(ereignisDtoList);
+            // save new ereignisse and override old ereignisse
+            SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_SET_EREIGNISSE);
+            val request = createPostWithBody(wahlbezirkID, ereignisseWriteDto);
+            api.perform(request).andExpect(status().isOk()).andReturn();
+            // read new ereignisse
+            SecurityUtils.runWith(Authorities.REPOSITORY_READ_EREIGNISSE);
+            val savedEreignisse = ereignisRepository.findByWahlbezirkID(wahlbezirkID);
+            List<Ereignis> expectedSavedEreignisse = new ArrayList<>();
+            for (EreignisDTO ereignisDto : ereignisseWriteDto.ereigniseintraege()) {
+                Ereignis ereignis = new Ereignis(wahlbezirkID, ereignisDto.beschreibung(), ereignisDto.uhrzeit(), ereignisDto.ereignisart());
+                expectedSavedEreignisse.add(ereignis);
+            }
```

Comment:
ist nicht spotless checked 😄 

```suggestion
            val expectedSavedEreignisse = ereignisseWriteDto.ereigniseintraege().stream()
                    .map(ereignisDto -> new Ereignis(wahlbezirkID, ereignisDto.beschreibung(), ereignisDto.uhrzeit(), ereignisDto.ereignisart()))
                    .toList();
```

Reply:
siehe commit [50cad96](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/50cad968e13e70c97c398f5338f1d8d08882e7df)